### DISCUSSION TITLE
<ScrollView Orientation="Both"> not working

### DISCUSSION THREAD

**Discussion by usa-** [dotnet/maui#7796] [Q&A]
I start VS 2022 version 17.3.0 preview 1.1 (the latest one).
I create a new “.NET MAUI App” project.
In the MainPage.xaml file, I change the code inside the ContentPage element to the following:

```
    <ScrollView Orientation="Both">
        <Image Source="dotnet_bot.png" HeightRequest="1000" WidthRequest="1000" Aspect="AspectFit" />
    </ScrollView>
```

In the code-behind file, MainPage.xaml.cs, I delete the OnCounterClicked function.
In the debug destination drop-down list, I select the Android Emulator.
I run the application.
Expected behavior: I am able to scroll the image both vertically and horizontally.
Actual behavior: I am able to scroll the image only horizontally.

**Comment by akhanalcs** [comment]
I believe you have to move this to Issues page. This is discussions page. 😊
You can move it to discussion by clicking 'Create Issue from discussion' link on the right hand side.

**Reply by usa-** [reply]
(palmface)
Thank you
